L1Trigger/L1TGlobal/src/CorrCondition.cc

```diff
@@ -1159,6 +1176,8 @@ const bool l1t::CorrCondition::evaluateCondition(const int bxEval) const {
         if (corrPar.corrCutType & 0x10)
           coshDeltaEtaPhy = 1.;
         double massSqPhy = et0Phy * et1Phy * (coshDeltaEtaPhy - cosDeltaPhiPhy);
+	if( corrPar.corrCutType & 0x40 ) // Added by R.Cavanaugh for displaced muons
```

Comment:
Please remove meaningless comments. Git versioning system keeps track of authors of commits.